languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt

```diff
@@ -95,3 +95,12 @@ Yahoo
 Zendesk
 dérapement
 dérapements
+Valachie
+d’Elne
+d’irais
+bage
+condiviser
+Caden
+SNSM
+monologale
+illocutionnaires
```

Comment:
Most these words are incorrect or in some cases rare proper names not worth adding.
The PR should be rejected.